deploy/central/tasks-chart/templates/task-ods-build-java.yml

```diff
@@ -0,0 +1,157 @@
+apiVersion: tekton.dev/v1beta1
+kind: ClusterTask
+metadata:
+  name: ods-build-java{{ .Values.taskSuffix }}
+spec:
+  description: |
+    Builds Java Gradle applications.
+
+    Note on build environment variables available:
+    - ODS_OUTPUT_DIR: this environment variable points to the folder
+    that this build expects generated application artifacts to be copied to.
+    The project gradle script should read this env var to copy all the
+    generated application artifacts.
+    - NEXUS_* env vars: following env vars NEXUS_HOST, NEXUS_USER and NEXUS_PASSWORD
+    are available and should be read by your gradle script.
+
+    The following steps are executed:
+    - build and test execution (using ./gradlew clean build)
+    - SonarQube quality scan
+
+    Tests exclude the vendor directory. Test results are converted into xUnit format.
+
+    Both xUnit report and coverage report are placed into .ods/artifacts.
+
+    To copy the generated application artifact e.g within the project gradle script follow this steps:
+    - read the env var 'ODS_OUTPUT_DIR' in the buildscript section of the gradle script:
+    buildscript {
+        ext
+                {
+                  outputDir = System.getenv('ODS_OUTPUT_DIR')
+                }
+    }
```

Comment:
Indentation looks off. Let's use a code block format, and 2 spaces as indentation consistently so that it is nice to read when rendered in Asciidoctor.